src/SocialServer/Network/Send.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Melia.Shared.Network;
+using Melia.Social.Database;
+
+namespace Melia.Social.Network
+{
+	public static class Send
+	{
+		/// <summary>
+		/// Login succeeded
+		/// </summary>
+		/// <param name="conn"></param>
+		public static void SC_LOGIN_OK(ISocialConnection conn)
+		{
+			var packet = new Packet(Op.SC_LOGIN_OK);
+
+			conn.Send(packet);
+		}
+
+		/// <summary>
+		/// Dummy
+		/// </summary>
+		public static void SC_FROM_INTEGRATE(ISocialConnection conn, int type)
+		{
+			var packet = new Packet(Op.SC_FROM_INTEGRATE);
+
+			switch (type)
+			{
+				case 0x01:
+					packet.PutLong(conn.Account.Id);
+					packet.PutInt(0);
+					break;
+				case 0x14:
+					packet.PutLong(conn.Account.Id);
+					packet.PutInt(0); // 728
+					break;
+			}
+			conn.Send(packet);
+		}
+
+		/// <summary>
+		/// Login related
+		/// </summary>
+		/// <param name="conn"></param>
+		public static void SC_NORMAL_Unknown_00(ISocialConnection conn)
+		{
+			var packet = new Packet(Op.SC_NORMAL);
+			packet.PutInt(NormalOp.Social.Unknown_00);
+
+			conn.Send(packet);
+		}
+
+		/// <summary>
+		/// Login related
+		/// </summary>
+		/// <param name="conn"></param>
+		public static void SC_NORMAL_Unknown_02(ISocialConnection conn)
+		{
+			var packet = new Packet(Op.SC_NORMAL);
+			packet.PutInt(NormalOp.Social.Unknown_02);
+
+			conn.Send(packet);
+		}
+
+		public static void SC_NORMAL_Unknown_0C(ISocialConnection conn)
+		{
+			var packet = new Packet(Op.SC_NORMAL);
+			packet.PutInt(NormalOp.Social.Buff_0C);
+
+			packet.PutByte(0);
+			packet.PutLong(554643486671500);
+			packet.PutByte(1);
+			packet.PutLong(conn.Account.Id);
+			packet.PutInt(4723);
+			packet.PutInt(1);
+			packet.PutInt(0);
+			packet.PutByte(0);
+
+			conn.Send(packet);
+		}
+
+		/// <summary>
+		/// Dummy
```

Comment:
Is it a dummy? To me that would indicate that it doesn't do anything, but that's clearly not the case. Could you elaborate on what these are?